session6/src/HomeworkS6/HomeworkS6.java

```diff
@@ -0,0 +1,28 @@
+package HomeworkS6;
+
+public class HomeworkS6 {
+    public static void main(String[] args) {
+
+        //Ex:1
+        String stringOne = "OpenAi";
+        String stringTwo = "OpenAi";
+        //System.out.println(stringOne == stringTwo);
+
+        //Ex:2
```

Comment:
each exercise should be decoupled. So, next time please make different methods and/or classes per each exercise.
In other words, please don't put all the exercises in a single `main` method.